client/src/client.rs

```diff
@@ -1507,6 +1507,11 @@ pub mod trigger {
     //! Module with queries for triggers
     use super::*;
 
+    /// Construct a query to get a trigger by its id
+    pub fn by_id(trigger_id: TriggerId) -> FindTriggerById {
+        FindTriggerById::new(trigger_id)
+    }
+
     /// Construct a query to get triggers by domain id
     pub fn by_domain_id(domain_id: DomainId) -> FindTriggersByDomainId {
```

Comment:
I don't think having this query makes sense anymore?

Reply:
Maybe we can have `FindTriggersByAccountId`?

Reply:
The query is still functional in the sense that it will find all triggers that are supposed to be executed on behalf of anyone belonging in the domain. It should at least be renamed to reflect what `DomainId` means (it's the authority's domain), if not removed as you suggest. And we can have something like `FindTriggersByAuthority`, which takes an `AccountId`.

Reply:
ok, I think we can keep `FindTriggersByDomainId`. But let's also add `FindTriggersByAccountId`. 

Instead of all this, we should have `FindTriggers` combined with a filter to select only for some `DomainId`/`AccountId`? @DCNick3 no?

Reply:
Yeah, that's what #3671 is about, but it's still some ways off..